### PR TITLE
[zh] Sync 1.18.2, 1.17.5, 1.16.7 release notes with support release, 1.16-eol and security-2023-003 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Translate 1.18.2, 1.17.5, 1.16.7 release notes, 1.16-eol and security-2023-003 into Chinese:
It is sync of https://github.com/istio/istio.io/pull/13619

```
content/zh/news/releases/1.16.x/announcing-1.16.7/index.md
content/zh/news/releases/1.17.x/announcing-1.17.5/index.md
content/zh/news/releases/1.18.x/announcing-1.18.2/index.md
content/zh/news/security/istio-security-2023-003/index.md
content/zh/news/support/announcing-1.16-eol-final/index.md
content/en/docs/releases/supported-releases/index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.16.x/announcing-1.16.7/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+本发布说明描述了 Istio 1.16.6 和 Istio 1.16.7 之间的不同之处。
+
+该版本为 Istio 1.16 的最后一个版本。
+
+{{< relnote >}}
+
+## 安全更新 {#security-update}
+
+- __CVE-2023-35941__:
+  (CVSS Score 8.6, High)：OAuth2 凭证具有永久有效性。
```
Comment (windsonsea) [inline content/zh/news/releases/1.16.x/announcing-1.16.7/index.md:22]:
```suggestion
  (CVSS Score 8.6, High)：OAuth2 凭证滥用永久有效性。
```

File: content/zh/news/releases/1.17.x/announcing-1.17.5/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+中阐述的安全漏洞。
+
+本发布说明描述了 Istio 1.17.4 和 Istio 1.17.5 之间的不同之处。
+
+{{< relnote >}}
+
+## 安全更新 {#security-update}
+
+- __CVE-2023-35941__:
+  (CVSS Score 8.6, High)：OAuth2 凭证具有永久有效性。
```
Comment (windsonsea) [inline content/zh/news/releases/1.17.x/announcing-1.17.5/index.md:20]:
```suggestion
  (CVSS Score 8.6, High)：OAuth2 凭证滥用永久有效性。
```

File: content/zh/news/releases/1.18.x/announcing-1.18.2/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+中阐述的安全漏洞。
+
+本发布说明描述了 Istio 1.18.1 和 Istio 1.18.2 之间的不同之处。
+
+{{< relnote >}}
+
+## 安全更新 {#security-update}
+
+- __CVE-2023-35941__:
+  (CVSS Score 8.6, High)：OAuth2 凭证具有永久有效性。
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18.2/index.md:20]:
```suggestion
  (CVSS Score 8.6, High)：OAuth2 凭证滥用永久有效性。
```

File: content/zh/news/releases/1.18.x/announcing-1.18.2/index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  (CVSS Score 6.5, Moderate)：由于侦听器耗尽而导致 gRPC 访问日志崩溃。
+- __CVE-2023-35943__:
+  (CVSS Score 6.3, Moderate)：删除原始头信息时，CORS 过滤器发生段错误。
+- __CVE-2023-35944__:
+  (CVSS Score 8.2, High)：Envoy 中大小写混合情况的 HTTP 请求和响应处理不正确。
+
+## 变更 {#changes}
+
+- **新增** 添加了对名为 `USE_EXTERNAL_WORKLOAD_SDS` 标志的支持。
+  当设置为 true 时，它将需要外部 SDS 工作负载套接字，并且如果未找到工作负载
```
Comment (windsonsea) [inline content/zh/news/releases/1.18.x/announcing-1.18.2/index.md:31]:
```suggestion
  当设置为 true 时，它将需要一个外部 SDS 工作负载套接字，如果找不到工作负载
```

File: content/zh/news/security/istio-security-2023-003/index.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+{{< security_bulletin >}}
+
+## CVE
+
+### Envoy CVE {#envoy-cves}
+
+- __CVE-2023-35941__:
+  (CVSS Score 8.6, High)：OAuth2 凭证具有永久有效性。
```
Comment (windsonsea) [inline content/zh/news/security/istio-security-2023-003/index.md:21]:
```suggestion
  (CVSS Score 8.6, High)：OAuth2 凭证滥用永久有效性。
```

File: content/zh/news/security/istio-security-2023-003/index.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+- __CVE-2023-35942__:
+  (CVSS Score 6.5, Moderate)：由于侦听器耗尽而导致 gRPC 访问日志崩溃。
+- __CVE-2023-35943__:
+  (CVSS Score 6.3, Moderate)：删除原始头信息时，CORS 过滤器发生段错误。
+- __CVE-2023-35944__:
+  (CVSS Score 8.2, High)：Envoy 中大小写混合情况的 HTTP 请求和响应处理不正确。
+
+## 我受到影响了吗？{#am-i-impacted}
+
+如果您接受来自不受信来源的 HTTP/2 流量，这将适用于大多数用户。
```
Comment (windsonsea) [inline content/zh/news/security/istio-security-2023-003/index.md:31]:
```suggestion
如果您接受来自不受信来源的 HTTP/2 流量，则您会受到影响，这种情况适用于大多数用户。
```

File: content/zh/news/security/istio-security-2023-003/index.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+如果您使用公共互联网上公开的网关，这一点尤其适用。
```
Comment (windsonsea) [inline content/zh/news/security/istio-security-2023-003/index.md:32]:
```suggestion
如果您使用公共互联网上公开的网关，要特别注意您可能已经受到了影响。
```

File: content/zh/docs/releases/supported-releases/index.md [hunk 8]
```diff
@@ -70,9 +70,9 @@ Istio 不保证超出支持窗口期的 Minor 版本都有已知的 CVE 补丁
 
 | Minor 版本 | 没有已知 CVE 的补丁版本                        |
 | ---------- | ------------------------------------------ |
-| 1.18.x     | 1.18.1+                                    |
-| 1.17.x     | 1.17.4+                                    |
-| 1.16.x     | 1.16.6+                                    |
+| 1.18.x           | 1.18.2+                                               |
```
[the remark this answers is not in the document]
Comment (wilsonwu) [reply content/zh/docs/releases/supported-releases/index.md]: Done

**Review by windsonsea — APPROVED** [review]
Thanks, LGTM

**Pull request merged**